Sources/MapboxNavigation/WayNameLabel.swift

```diff
@@ -21,25 +21,26 @@ open class WayNameLabel: StylableLabel {
     @objc dynamic public var roadShieldDefaultColor: UIColor = .roadShieldDefaultColor
     
     // When the map style changes, update the sprite repository and the label.
-    func updateStyle(styleURI: StyleURI?) {
-        spriteRepository.updateStyle(styleURI: styleURI) { [weak self] _ in
-            guard let self = self,
-                  let roadName = self.text else { return }
+    func updateStyle(styleURI: StyleURI?, idiom: UIUserInterfaceIdiom = .phone) {
+        spriteRepository.updateStyle(styleURI: styleURI, idiom: idiom) { [weak self] _ in
```

Comment:
Do we really need to wait for completion handler to call `self.setup(with: roadName, idiom: idiom)`?

Reply:
I think so, in case there's no road name update.